projects/pgai/tests/test_model_download.py

```diff
@@ -0,0 +1,63 @@
+import shutil
+import tempfile
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+
+
+def test_download_models():
+    """Test that download-models command works correctly."""
+    # Create a temporary directory for the test
+    with patch("docling.utils.model_downloader.download_models") as mock_download:
+        from pgai.cli import download_models
+        from pgai.vectorizer.parsing import DOCLING_CACHE_DIR
+
+        # Run the CLI command
+        result = CliRunner().invoke(download_models)
+
+        # Check that the command executed successfully
+        assert result.exit_code == 0
+
+        # Verify that download_models was called with the correct parameters
+        mock_download.assert_called_once_with(
+            progress=True,
+            output_dir=DOCLING_CACHE_DIR,
+        )
+
+
+@pytest.mark.skip("Integration test")
```

Comment:
why is this skipped?

Reply:
Because that downloads gigabytes of data and runs for minutes. I didn't want to block CI for that long all the time. There is a mocked version above that runs always.

Reply:
👍